src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -657,9 +651,13 @@ std::string UsgsAstroFrameSensorModel::getModelState() const {
                                    m_currentParameterValue[2], m_currentParameterValue[3],
                                    m_currentParameterValue[4], m_currentParameterValue[5],
                                    m_currentParameterValue[6]}},
-      {"m_currentParameterCovariance", m_currentParameterCovariance}
+      {"m_imageIdentifier", m_imageIdentifier},
+      {"m_collectionIdentifier", m_collectionIdentifier}
     };
-
+    state["m_referencePointXyz"] = json();
+    state["m_referencePointXyz"]["x"] = m_referencePointXyz.x;
+    state["m_referencePointXyz"]["y"] = m_referencePointXyz.y;
+    state["m_referencePointXyz"]["z"] = m_referencePointXyz.z;
```

Comment:
In `contructStateFromIsd` this is being initialized as a `std::vector<float>`, probably best to stick with the vector since that's how most of these values are being handled and only translate to the point object when we have to, (i.e. in `getReferencePoint()`)